pyproject.toml

```diff
@@ -306,7 +306,8 @@ all = [
 ]
 
 [tool.poetry.dev-dependencies]
-# We pin black so that our tests don't start failing on new releases.
+# We pin black via the lock file so that our tests don't start failing on new
+# releases. We use a >= comparison ro receive updates from dependabot.
```

Comment:
As per the thread: I think I mislead you on this. Dependabot does propose updates for packages pinned in pyproject.toml; e.g. I'm pretty sure it's done that for `ruff` at some point in the past. We could probably get away with `*` (as we seem to do with mypy) tbh.

Suggested words:

> We pin development dependencies in poetry.lock so that our tests don't start failing on new releases. Keeping lower bounds loose here means that dependabot can bump versions without having to update the `content-hash` in the lockfile. This helps prevents merge conflicts when running a batch of dependabot updates.